pages/common/cpio.md

```diff
@@ -1,16 +1,15 @@
 # cpio
 
-> Archiving utility.
+> Copies files to and from archives.
 
-- Create an archive from files
+- Take a list of file names from standard input and creates an archive from them:
 
-`echo "{{file1}} {{file2}} {{file3}}" | cpio -ov > {{archive.cpio}}`
+`echo "{{file1}} {{file2}} {{file3}}" | cpio -o > {{[o]utput_archive.cpio}}`
```

Comment:
I was slightly confused by the .cpio extension, as I had never heard of an archive format by that name. I understood it after reading the project page (emphasis mine):

> GNU **cpio supports the following archive formats: binary, old ASCII, new ASCII, crc, HPUX binary, HPUX old ASCII, old tar, and POSIX.1 tar.** The tar format is provided for compatability with the tar program. By default, cpio creates binary format archives, for compatibility with older cpio programs. **When extracting from archives, cpio automatically recognizes which kind of archive it is reading** and can read archives created on machines with a different byte-order.

I think we should incorporate that information into this description.